x/reward/client/cli/tx.go

```diff
@@ -0,0 +1,227 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/provenance-io/provenance/x/reward/types"
+)
+
+// Flag names and values
+const (
+	FlagTotalRewardPool         = "total-reward-pool"
+	FlagMaxRewardByAddress      = "max-reward-by-address"
+	FlagStartTime               = "start-time"
+	FlagClaimPeriods            = "claim-periods"
+	FlagClaimPeriodDays         = "claim-period-days"
+	FlagExpireDays              = "expire-days"
+	FlagQualifyingActions       = "qualifying-actions"
+	FlagMaxRolloverClaimPeriods = "max-rollover-periods"
+)
+
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Aliases:                    []string{"r"},
+		Short:                      "Transaction commands for the reward module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		GetCmdRewardProgramAdd(),
+		GetCmdEndRewardProgram(),
+		GetCmdClaimReward(),
+	)
+
+	return txCmd
+}
+
+func GetCmdRewardProgramAdd() *cobra.Command {
+	actionsExampleJSON := "{\"qualifying_actions\":[{\"delegate\":{\"minimum_actions\":\"0\",\"maximum_actions\":\"0\",\"minimum_delegation_amount\":{\"denom\":\"nhash\",\"amount\":\"0\"},\"maximum_delegation_amount\":{\"denom\":\"nhash\",\"amount\":\"100\"},\"minimum_active_stake_percentile\":\"0.000000000000000000\",\"maximum_active_stake_percentile\":\"1.000000000000000000\"}}]}"
```

Comment:
If you delimited this with ticks, you wouldn't have to escape all the quotes.

Alternatively, since it's only used in one place, you could just put it there instead of `%s`ing it in.